docs/src/pages/components/timeline/SimpleTimeline.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Timeline from '@material-ui/lab/Timeline';
+import TimelineItem from '@material-ui/lab/TimelineItem';
+import TimelineItemTail from '@material-ui/lab/TimelineItemTail';
```

Comment:
Should `TimelineItemTail` be part of the public API? For comparison, we didn't include it in the Stepper component.

Reply:
What do you think of calling the tail a connector (using Stepper terminology)?